share/lib/python/neuron/tests/_subclass.py

```diff
@@ -14,7 +14,11 @@
 endtemplate A
 ''')
 
-class A1(hclass(h.A)) :
+_cls = hclass(h.A)
+class A1(_cls) :
+  def __new__(cls, arg):
```

Comment:
Is explicitly invoking `__new__` required when subclassing using your `hclass`? Isn't that less convenient? And certainly it's not a thing one normally does when subclassing Python classes.